.kokoro

```diff
@@ -227,7 +227,7 @@ run_cocoapods() {
         bash scripts/test_all catalog/MDCCatalog.xcworkspace:MDCUnitTests
       fi
     elif [ "$DEPENDENCY_SYSTEM" = "cocoapods-podspec" ]; then
-      pod lib lint MaterialComponents.podspec
+      pod lib lint MaterialComponents.podspec --verbose
```

Comment:
Should we add MaterialComponentsAlpha.podspec?

Reply:
Linting is a requirement for pod publication, so because we don't publish the Alpha spec we also don't need to lint it. Our build jobs sufficiently verify that the pod is working via as expected for day-to-day workflows with `pod install`.